--- gcc-4.9.2/gcc/config.gcc	2014-09-17 16:16:02.000000000 +0200
+++ config.gcc	2014-11-17 11:11:43.381317709 +0100
@@ -616,6 +616,12 @@
 
 # Common parts for widely ported systems.
 case ${target} in
+*-*-trinix*)
+  extra_parts="crtbegin.o crtend.o"
+  gas=yes
+  gnu_ld=yes
+  default_use_cxa_atexit=yes
+  ;;
 *-*-darwin*)
   tmake_file="t-darwin ${cpu_type}/t-darwin"
   tm_file="${tm_file} darwin.h"
@@ -1166,6 +1172,11 @@
 	         gnu-user.h linux.h glibc-stdint.h frv/linux.h"
 	tmake_file="${tmake_file} frv/t-frv frv/t-linux"
 	;;
+x86_64-*-trinix*)
+	tm_file="${tm_file} i386/unix.h i386/att.h dbxelf.h elfos.h i386/i386elf.h i386/x86-64.h trinix.h"
+	tmake_file="i386/t-i386elf i386/t-crtstuff t-svr4"
+	use_fixproto=yes
+	;;
 moxie-*-elf)
 	gas=yes
 	gnu_ld=yes
