--- binutils-2.24/ld/Makefile.am	2013-11-26 12:37:33.000000000 +0100
+++ Makefile.am	2014-11-15 19:34:51.169884400 +0100
@@ -480,6 +480,7 @@
 	eelf32btsmip_fbsd.c \
 	eelf32btsmipn32.c \
 	eelf32btsmipn32_fbsd.c \
+	eelf_x86_64_trinix.c \
 	eelf32ebmip.c \
 	eelf32ebmipvxworks.c \
 	eelf32elmip.c \
@@ -2138,6 +2139,9 @@
 eelf_x86_64.c: $(srcdir)/emulparams/elf_x86_64.sh \
   $(ELF_DEPS) $(srcdir)/scripttempl/elf.sc ${GEN_DEPENDS}
 	${GENSCRIPTS} elf_x86_64 "$(tdir_elf_x86_64)"
+eelf_x86_64_trinix.c: $(srcdir)/emulparams/elf_x86_64_trinix.sh \
+ $(ELF_DEPS) $(srcdir)/scripttempl/elf.sc ${GEN_DEPENDS}
+	${GENSCRIPTS} elf_x86_64_trinix "$(tdir_elf_x86_64_trinix)"
 eelf_x86_64_fbsd.c: $(srcdir)/emulparams/elf_x86_64_fbsd.sh \
   $(srcdir)/emulparams/elf_x86_64.sh \
   $(ELF_DEPS) $(srcdir)/scripttempl/elf.sc ${GEN_DEPENDS}
