--- gcc-4.9.2/config.sub	2013-10-01 18:50:56.000000000 +0200
+++ config.sub	2014-11-17 10:49:33.812598932 +0100
@@ -1357,6 +1357,7 @@
 	      | -sym* | -kopensolaris* | -plan9* \
 	      | -amigaos* | -amigados* | -msdos* | -newsos* | -unicos* | -aof* \
 	      | -aos* | -aros* \
+		  | -trinix* \
 	      | -nindy* | -vxsim* | -vxworks* | -ebmon* | -hms* | -mvs* \
 	      | -clix* | -riscos* | -uniplus* | -iris* | -rtu* | -xenix* \
 	      | -hiux* | -386bsd* | -knetbsd* | -mirbsd* | -netbsd* \
